packages/dd-trace/test/appsec/reporter.spec.js

```diff
@@ -119,6 +125,13 @@ describe('reporter', () => {
 
       expect(telemetry.incrementWafInitMetric).to.have.been.calledOnceWithExactly(wafVersion, rulesVersion)
     })
+
+    it('should add _dd.p.appsec entrie if standalone ASM enabled', () => {
+      isStandaloneEnabled.returns(true)
+
+      Reporter.reportWafInit(wafVersion, rulesVersion, diagnosticsRules)
+      expect(Reporter.metricsQueue.get('_dd.p.appsec')).to.be.eq(1)
```

Comment:
I miss a test checking that `_dd.p.appsec` is not sent when `isStandaloneEnabled` is `false`